src/app/BreedProcessor.ts

```diff
@@ -16,13 +16,11 @@ export class BreedProcessor {
     this.randomResultInterfaceFox = new FirstDice();
   }
 
-  processBreedPhase({ theHerd }: Player): void {
-    const wolf = this.getRandomResult(
-      this.randomResultInterfaceWolf.getRandomValue(),
-    );
-    const fox = this.getRandomResult(
-      this.randomResultInterfaceFox.getRandomValue(),
-    );
+  processBreedPhase({ theHerd }: Player): [AnimalNames, AnimalNames] {
+    const wolfResult = this.randomResultInterfaceWolf.getRandomValue();
+    const wolf = this.getRandomResult(wolfResult);
```

Comment:
I am not quite sure I understand wolf and wolfResult. Consider name changing, it is unclear what is does from variables' names

Reply:
It was already there in @Enessetere BreedProcessor.

Reply:
but it's about distinguishing dice one form another